Autocoders/Python/doc/tlmLayout/LayoutExampleTlmLayoutAi.csv

```diff
@@ -1 +1,1142 @@
Packet,,,,,
Identifier,U64packet,205,,,
,,,,,
Header,ID,,,8,
Header,Sequence,,,8,
Header,Time,,,,
,,,,,
Comment,Mnemonic,Tlm DB identifier,Tlm DB data type,Bits,Comment / description
item,U64,12317,U64,64,TOTALISFTELMMSG1 / 0x301D
item,U64,12318,U64,64,TOTALISFTELMMSG2 / 0x301E
item,U64,12319,U64,64,TOTALISFTELMMSG3 / 0x301F
item,U64,12320,U64,64,TOTALISFTELMMSG4 / 0x3020
item,U64,12321,U64,64,TOTALISFTELMMSG5 / 0x3021
item,U64,12322,U64,64,TOTALISFTELMMSG6 / 0x3022
,,,,,
Comment,Packet with 6 U64s for Telemetry Collector Stats,#40,,,
Packet,,,,,
Identifier,U64packet,206,,,
,,,,,
Header,ID,,,8,
Header,Sequence,,,8,
Header,Time,,,,
,,,,,
Comment,Mnemonic,Tlm DB identifier,Tlm DB data type,Bits,Comment / description
item,U64,12323,U64,64,TOTALISFTELMMSG7 / 0x3023
item,U64,12324,U64,64,TOTALISFTELMMSG8 / 0x3024
item,U64,12325,U64,64,TOTALISFTELMMSG9 / 0x3025
item,U64,12326,U64,64,TOTALISFTELMMSG10 / 0x3026
item,U64,12327,U64,64,TOTALISFTELMMSG11 / 0x3027
item,U64,12328,U64,64,TOTALISFTELMMSG12 / 0x3028
,,,,,
Comment,Packet with 6 U64s for Telemetry Collector Stats,#41,,,
Packet,,,,,
Identifier,U64packet,207,,,
,,,,,
Header,ID,,,8,
Header,Sequence,,,8,
Header,Time,,,,
,,,,,
Comment,Mnemonic,Tlm DB identifier,Tlm DB data type,Bits,Comment / description
item,U64,12329,U64,64,TOTALISFTELMMSG13 / 0x3029
item,U64,12330,U64,64,TOTALISFTELMMSG14 / 0x302A
item,U64,12331,U64,64,TOTALISFTELMMSG15 / 0x302B
item,U64,12332,U64,64,TOTALISFTELMMSG16 / 0x302C
item,U64,12333,U64,64,TOTALISFTELMMSG17 / 0x302D
item,U64,12334,U64,64,TOTALISFTELMMSG18 / 0x302E
,,,,,
Comment,Packet with 6 U64s for Telemetry Collector Stats,#42,,,
Packet,,,,,
Identifier,U64packet,208,,,
,,,,,
Header,ID,,,8,
Header,Sequence,,,8,
Header,Time,,,,
,,,,,
Comment,Mnemonic,Tlm DB identifier,Tlm DB data type,Bits,Comment / description
item,U64,12335,U64,64,TOTALISFTELMMSG19 / 0x302F
item,U64,12336,U64,64,TOTALISFTELMMSG20 / 0x3030
item,U64,12337,U64,64,TOTALISFTELMMSG21 / 0x3031
item,U64,12338,U64,64,TOTALISFTELMMSG22 / 0x3032
item,U64,12339,U64,64,TOTALISFTELMMSG23 / 0x3033
item,U64,12340,U64,64,TOTALISFTELMMSG24 / 0x3034
,,,,,
Comment,Packet with 5 U64s for Telemetry Collector Stats,#43,,,
Packet,,,,,
Identifier,U64packet,209,,,
,,,,,
Header,ID,,,8,
Header,Sequence,,,8,
Header,Time,,,,
,,,,,
Comment,Mnemonic,Tlm DB identifier,Tlm DB data type,Bits,Comment / description
item,U64,12341,U64,64,TOTALISFTELMMSG25 / 0x3035
item,U64,12342,U64,64,TOTALISFTELMMSG26 / 0x3036
item,U64,12343,U64,64,TOTALISFTELMMSG27 / 0x3037
item,U64,12344,U64,64,TOTALISFTELMMSG28 / 0x3038
item,U64,12345,U64,64,TOTALISFTELMMSG29 / 0x3039
,,,,,
Comment,Packet with3 U64s for Telemetry Collector Stats,#44,,,
Packet,,,,,
Identifier,U64packet,210,,,
,,,,,
Header,ID,,,8,
Header,Sequence,,,8,
Header,Time,,,,
,,,,,
Comment,Mnemonic,Tlm DB identifier,Tlm DB data type,Bits,Comment / description
item,U64,12346,U64,64,TOTALISFLOGEVENTMSG / 0x303A
item,U64,12347,U64,64,TOTALQUEUEFULL / 0x303B
item,U64,12348,U64,64,TOTALFF3MSG / 0X303C
```

Comment:
This was an interesting (and longstanding) bug in check-spelling... It mishandled mixed line-ending files and suggested ignoring this file. Newer versions of check-spelling don't suggest ignoring this kind of file which led me to investigate what was so special about it...

I can't see any particular reason that this file had mixed line-endings, so I'm converting it to standard line endings which means that its easier for GitHub and git tools to view/parse.